js/ui/components/clubhouse.js

```diff
@@ -211,5 +367,9 @@ var ClubhouseComponent = new Lang.Class({
         if (this.proxy.g_name_owner)
             this._clubhouseSource.activateAction('stop-quest', null);
     },
+
+    _onVisibilityChanged: function() {
+        this.parent();
+    },
```

Comment:
This does not do anything and can be removed

Reply:
Sorry. I thought I had removed that before pushing the last version...